catalogue_graph/src/ingestor/models/denormalised/work.py

```diff
@@ -0,0 +1,76 @@
+from datetime import datetime
+from typing import Literal
+
+from pydantic import BaseModel, field_validator
+
+from ingestor.models.shared.concept import Concept, Contributor, Genre, Subject
+from ingestor.models.shared.holdings import Holdings
+from ingestor.models.shared.id_label import Id, IdLabel
+from ingestor.models.shared.identifier import (
+    SourceIdentifier,
+)
+from ingestor.models.shared.image import ImageData
+from ingestor.models.shared.item import Item
+from ingestor.models.shared.location import DigitalLocation
+from ingestor.models.shared.note import Note
+from ingestor.models.shared.production import ProductionEvent
+from ingestor.models.shared.serialisable import ElasticsearchModel
+from utils.types import WorkType
+
+
+class CollectionPath(BaseModel):
+    path: str
+    label: str | None = None
+
+
+class DenormalisedWorkData(ElasticsearchModel):
```

Comment:
Noting that there will eventually be some overlap between the work model being created here, and the necessary post-transform work model from the EBSCO adapter (and future adapter) steps.